src/components/results/response/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classnames from 'classnames';
+import RequestHeader from '../header';
+import { TREE_VIEW } from '../results-view-selector';
+import RequestBody from '../body';
+
+import './style.css';
+
+class Index extends React.Component {
+	componentWillMount() {
+		this.setState( { view: TREE_VIEW } );
+	}
```

Comment:
do need to call this on mount? I think we should be fine setting the state to its initial value, which would eliminate a temporal dead zone where no `view` exists on the component's state.

```ts

class Index extends React.Component {
	state = {
		view: TREE_VIEW
	};

	onViewChange = view => this.setState( { view } );

	render() {
		…
	}
}
```